src/index.html

```diff
@@ -1,14 +1,97 @@
 <!doctype html>
 <html lang="en">
+
   <head>
     <meta charset="UTF-8">
-    <meta name="viewport"
-          content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
+    <meta name="viewport" content="width=device-width, user-scalable=no,
+          initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>Moyo header</title>
     <link rel="stylesheet" href="./style.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500&display=swap"
+    rel="stylesheet"
+    >
+
   </head>
-  <body>
-    <h1>Moyo header</h1>
+
+  <body id="home">
+    <header class="topBar">
+      <a
+        href="#home"
+        class="navLogoLink"
+      >
+        <img
+          src="images/logo.png"
+          alt="MOYO logo"
+          class="navLogo"
+        >
+      </a>
+
+      <nav class="navBar">
+        <ul class="navList">
+          <li class="navItem">
+            <a
+              href="#home"
+              class="is-active navLink"
+              >Apple
+            </a>
```

Comment:
```suggestion
            <a href="#home" class="is-active navLink">
              Apple
            </a>
```
якщо менше трьох атрибутів і дозволяє довжина:
або 
<a href="#home" class="is-active navLink">
  Apple
</a>

або (якщо більше трьох)
<a
  href="#home"
  class="is-active navLink"
>
  Apple
</a>

виправ у всьому файлі